src/auto/gesture_stylus.rs

```diff
@@ -0,0 +1,184 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+#[cfg(any(feature = "v3_24", feature = "dox"))]
+use gdk;
+#[cfg(any(feature = "v3_24", feature = "dox"))]
+use glib::object::Cast;
+#[cfg(any(feature = "v3_24", feature = "dox"))]
+use glib::object::IsA;
+use glib::object::ObjectType as ObjectType_;
+use glib::signal::connect_raw;
+use glib::signal::SignalHandlerId;
+use glib::translate::*;
+use glib_sys;
+use gtk_sys;
+use libc;
+use std::boxed::Box as Box_;
+use std::fmt;
+#[cfg(any(feature = "v3_24", feature = "dox"))]
+use std::mem;
+use std::mem::transmute;
+use EventController;
+use Gesture;
+use GestureSingle;
+#[cfg(any(feature = "v3_24", feature = "dox"))]
+use Widget;
+
+glib_wrapper! {
+    pub struct GestureStylus(Object<gtk_sys::GtkGestureStylus, gtk_sys::GtkGestureStylusClass, GestureStylusClass>) @extends GestureSingle, Gesture, EventController;
+
+    match fn {
+        get_type => || gtk_sys::gtk_gesture_stylus_get_type(),
+    }
+}
+
+impl GestureStylus {
+    #[cfg(any(feature = "v3_24", feature = "dox"))]
+    pub fn new<P: IsA<Widget>>(widget: &P) -> GestureStylus {
+        skip_assert_initialized!();
+        unsafe {
+            Gesture::from_glib_full(gtk_sys::gtk_gesture_stylus_new(
+                widget.as_ref().to_glib_none().0,
+            ))
+            .unsafe_cast()
+        }
+    }
+
+    //#[cfg(any(feature = "v3_24", feature = "dox"))]
```

Comment:
@sdroege any idea how can I generate this one? :)

Reply:
You have to do it manually, see the docs of this function in C: `axes: array of requested axes, terminated with GDK_AXIS_IGNORE`